index.js

```diff
@@ -5,5 +5,8 @@
 module.exports = function(content) {
 	this.cacheable && this.cacheable();
 	this.value = content;
-	return "module.exports = " + JSON.stringify(content);
+	content = JSON.stringify(content)
+		.replace(/\u2028/g, '\\u2028')
+		.replace(/\u2029/g, '\\u2029');
```

Comment:
Alternatively, use `jsesc` if you want more control over the output (e.g. if you want completely ASCII-safe output).